apps/rush-lib/src/logic/UnlinkManager.ts

```diff
@@ -61,12 +61,10 @@ export class UnlinkManager {
         didDeleteAnything = true;
       }
 
-      const projectDependencyManifestFilePath: string = PnpmProjectDependencyManifest.getFilePathForProject(
-        rushProject
-      );
-      if (FileSystem.exists(projectDependencyManifestFilePath)) {
-        console.log(`Deleting ${projectDependencyManifestFilePath}`);
-        FileSystem.deleteFile(projectDependencyManifestFilePath);
+      const projectShrinkwrapFilePath: string = BaseProjectShrinkwrapFile.getFilePathForProject(rushProject);
+      if (FileSystem.exists(projectShrinkwrapFilePath)) {
+        console.log(`Deleting ${projectShrinkwrapFilePath}`);
+        FileSystem.deleteFile(projectShrinkwrapFilePath);
```

Comment:
Couldn't you use `deleteIfExistsAsync`?

Reply:
We don't have a shrinkwrap at this point, which is required to create the `ProjectShrinkwrap`. Since we only need the path here and the shrinkwrap would just get thrown away, I'm going to leave this as-is. This will go away when the linking phase is removed with legacy Rush installs as well, so it will be short-lived.